accession-commons-jpa/src/test/resources/features/modification.feature

```diff
@@ -0,0 +1,40 @@
+Feature: Accession modification
+
+  Scenario Outline: I want to create a new patch of an existing object, in order to register improvements made to it
+  without losing access to the old information.
+    Given already accessioned <objects>
+    When user sends patch <input> for accession <accession>
+    Then user should receive version list for <accession> with 2 version
+    And patch must have versions increased
+    Examples:
+      | objects | input | accession    |
+      | A       | AA    | id-service-A |
+      | B       | BB    | id-service-B |
+      | A,B,C,E | AA    | id-service-A |
+
+  Scenario: I want to create a new patch of a non existing object
+    When user sends patch AA for accession id-service-A
+    Then user receives accession does not exist
+
+  Scenario Outline: I want to update the fields that identify uniquely an object.
+  without losing access to the old information.
+    Given already accessioned <objects>
+    When user updates <accession> patch <patch> with <input>
```

Comment:
can we have <accession> version <version> instead of "patch" please, to match with "Then" output

Reply:
I am more keen to use patch in both that version.

Reply:
Agree with favoring "patch", it is the term we have agreed to use. "version" is just the way the field is serialized into the database because there is no way around it.